### ISSUE TITLE
RGB/Correct resolution output

### ISSUE THREAD

**Issue by retrorepair** [fhoedemakers/pico-infonesPlus#73]
Just wondered if there were any plans for either of these?

Of course RGB direct output would mean a lot more GPIO and extra hardware (resistors) but if the original resolutions/refresh rates were supported a cheap HDMI to VGA DAC could be used to get RGB output for CRTs.

**Comment by fhoedemakers** [comment]
No no plans for this.  Looked shortly into https://shop.pimoroni.com/products/pimoroni-pico-vga-demo-base but did nothing with it.

**Comment by retrorepair** [comment]
Fair enough. I'd imagine supporting the correct resolutions wouldn't be too difficult though as an option? I imagine they're currently integer scaled?

I may have a play with it if I can work out how to compile it :)

**Comment by fhoedemakers** [comment]
There are build scripts included on the repo. They are documented near the end of the README. The scripts are intended to run on Linux on a raspberry pi. You can also build on x86 using Ubuntu in a VM. I now develop using a Windows 11 desktop with Ubuntu running in WSL

**Comment by retrorepair** [comment]
Great thanks. Will get that set up and have a fiddle with it
